src/main/java/core/basesyntax/StorageImpl.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int INITIAL_CAPACITY = 10;
+    private Object[] keys = new Object[INITIAL_CAPACITY];
```

Comment:
```suggestion
    private K[] keys;
```

initialise keys and values in constructor